_posts/2013-04-02-practices-windows.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: post
+title: "Windows"
+category: user-guide
+permalink: windows
+github: "https://github.com/stefanpenner/ember-cli/blob/gh-pages/_posts/2013-04-02-practices-windows.md"
+---
+
+Without any configuration build times on Windows are longer than on Linux or Mac OS X. 
+
+* Much of that penalty is not because of node or ember-cli, but because of Windows services monitoring your filesystem. See the section below for how to disable those.
```

Comment:
@felixrieseberg I assume your `ember-cli` comment is referring to the line above. I agree with what @dukex said in https://github.com/ember-cli/ember-cli/pull/4738#issuecomment-138148678 that an even more correct way is `Ember CLI`. It is also consistent with e.g. http://www.ember-cli.com/user-guide/#getting-started so I will change to that unless @stefanpenner benevolently :wink: can guide into yet another direction.

For consistency I will also change `node` into `Node` which is the way it's written elsewhere in the Ember CLI guide.